libs/langchain-aws/src/types.ts

```diff
@@ -19,3 +19,26 @@ export type BedrockToolChoice =
   | ToolChoice.AutoMember
   | ToolChoice.ToolMember;
 export type ChatBedrockConverseToolType = BindToolsInput | BedrockTool;
+
+export type MessageContentReasoningBlockReasoningText = {
+  type: "reasoningContent";
```

Comment:
These are our internal representations?

Why don't we just preserve the ones returned from Bedrock?

Reply:
Ok - only other concern then is to try to keep it inline with changes in Python as well

Reply:
Not sure who's picking up the python one, but we're consciously _not_ forcing structural similarity between the two libs anymore unless there's a good reason to (e.g. they need to interop). Winds up causing too many pythonic conventions that feel weird in JS